### PR TITLE
Add .editorconfig.

### PR DESCRIPTION
**Pull request by woodruffw**
I mostly copied from the EditorConfig [example](http://editorconfig.org/#example-file), so tell me if I missed anything :smile:.

cc @UniqMartin

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```
Comment (UniqMartin) [inline .editorconfig]: I would additionally add `trim_trailing_whitespace = true` here.

**Comment by UniqMartin** [comment]
One addition, otherwise I'm happy with the adapted example. 😸

Just out of curiosity, what are you actually using as your programmer's editor of choice?

**Comment by woodruffw** [comment]
I currently use Sublime Text 3 for most of my development, and I occasionally switch back to my original editor (gvim). Most of `ruby-macho` was written in ST, though :smile:.

**Comment by UniqMartin** [comment]
Nice! I'm also predominantly a Sublime Text user. :+1:

**Pull request merged**